### PR TITLE
Zuul: test deployment job

### PR DESCRIPTION
**Pull request by jpopelka**

### CODE REVIEW

File: .zuul.yaml [hunk 1]
```diff
@@ -3,6 +3,22 @@
     check:
       jobs:
         - pre-commit
+        - test
     gate:
       jobs:
         - pre-commit
+
+- job:
+    name: test
```
Comment (TomasTomecek) [inline .zuul.yaml:12]: names are global, so please make it more unique, something like `deployment-tests`

**Review by TomasTomecek — CHANGES_REQUESTED** [review]
otherwise LGTM, nicely done

File: .zuul.yaml (hunk 1 above)
Comment (jpopelka) [reply .zuul.yaml]: I never remember that, thanks.

**Review by TomasTomecek — APPROVED** [review]
LGTM

did Lentilka help you?

**Comment by jpopelka** [comment]
:dog: Yes, she did, but it was expensive :candy: :candy:  :candy: :meat_on_bone: :cut_of_meat:

**Pull request merged**